config/build/rollup.config.js

```diff
@@ -200,77 +149,30 @@ const createConfig = (
  * each package. If the package has a `browser` field, then we generate
  * browser and node assets. If not, we just generate the node assets.
  * Note that we also get the output paths from the package.json.
- *
- * We also can filter the outputs based on command line options:
- * `--configPlatforms` - Comma-separated list. Valid values are "browser"
- *                       and "node".
- * `--configFormats`   - Comma-separated list. Valid values are "cjs" and
- *                       "esm". If not specified, then we generate both.
  */
-const getPackageInfo = (commandLineArgs, pkgName) => {
+const getPackageInfo = (pkgName) => {
     const pkgJsonPath = makePackageBasedPath(pkgName, "./package.json");
     if (!fs.existsSync(pkgJsonPath)) {
         return [];
     }
     const pkgJson = JSON.parse(fs.readFileSync(pkgJsonPath));
 
-    // Determine what formats and platforms we are building.
-    const formats = getFormats(commandLineArgs);
-
     const configs = [];
 
-    if (pkgJson.exports) {
-        for (const exportConfig of Object.values(pkgJson.exports)) {
-            if (exportConfig.require && formats.has("cjs")) {
-                configs.push({
-                    name: pkgName,
-                    fullName: pkgJson.name,
-                    version: pkgJson.version,
-                    format: "cjs",
-                    platform: "browser",
-                    inputFile: exportConfig.source,
-                    file: exportConfig.require,
-                    plugins: [],
-                });
-            }
+    if (!pkgJson.exports) {
+        console.log(pkgName);
```

Comment:
We don't need anything else here? Just log the name?

Reply:
Thanks for catching this. I've changed it to throw. We don't want to allow older-style package.json's when we build/publish.